File: problems/mario/mario.c
#include <stdio.h>
#include <cs50.h>

int main(void)
{
    int height;
    //Prompt user for height
    do
    {
        height = get_int("Height : ");
    }
    while ((height <= 0) || (height >= 9));
    
    //For every row
    for (int i = 1; i <= height; i++)
    {   
        //How many spaces first
        for (int j = 0; j < height - i; j++)
        {
            printf(" ");
        }
        //How many hashes
        for (int j = 0; j < i; j++)
        {
            printf("#");
        }
        //THE 2 spaces GAP 
        for (int j = 0; j < 2; j++)
        {
            printf(" ");
        }
        //Last hash row
        for (int j = 0; j < i; j++)
        {
            printf("#");
        }
        printf("\n");
    }

}

File: problems/sentimental/mario/mario.py
# Import get_string from cs50 library
from cs50 import get_int
# Prompt user for input
while True:
    height = get_int("Height: ")
    # Get out of the infinite loop if Height is between 1 and 8 inclusive
    if height in range(1, 9):
        break
# How many floors
for i in range(1, height + 1):
    # How many spaces first
    for j in range(0, height - i):
        print(" ", end="")
    # How many hashes    
    for j in range(0, i):
        print("#", end="")
    print("  ", end="")
    # Last hash row
    for j in range(0, i):
        print("#", end="")
    print()